corehq/form_processor/sql_functions/get_case_form_ids.sql

```diff
@@ -0,0 +1,16 @@
+DROP FUNCTION IF EXISTS get_case_form_ids(text);
+
+CREATE FUNCTION get_case_form_ids(case_id text) RETURNS TABLE (form_id VARCHAR(255)) AS $$
```

Comment:
I don't not a whole lot about pgsql, but I believe it's possible to use `SETOF text` or something like that for a function return type, which I think would achieve the same thing as returning the table here. Do you have any idea if one approach is more performant than the other? My intuition is that returning a table is a more heavy duty operation...